lib/caracal/core/models/table_model.rb

```diff
@@ -99,8 +101,33 @@ def cell_style(models, options={})
             m.apply_styles(options)
           end  
         end
+
+        # This method allows a user to specify one or more
+        # rows that should be rendered with that
```

Comment:
Supernit: `with that` reads a little goofy

Reply:
totally